main/src/main/java/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1195,6 +1200,54 @@ private void dropGeneratedWaypoints() {
         }
     }
 
+    private List<Waypoint> getSortedWaypointList(final Geocache cache) {
```

Comment:
Good call to make this a method. I am surprised it was not before.
But maybe move it to class Geocache or as helper method to class Waypoint?